sdk/core/azure-core/src/http/retry_policy.cpp

```diff
@@ -214,10 +214,10 @@ bool RetryPolicy::ShouldRetryOnResponse(
     auto const sc = response.GetStatusCode();
     if (statusCodes.find(sc) == statusCodes.end())
     {
-      if (Log::ShouldWrite(Logger::Level::Warning))
```

Comment:
Does this change only impact HTTP status code 200? It seems like it might impact other status codes as well.

Does that mean we want to have this as informational for any http status codes that we don't retry on?

Reply:
Ah I see, we have the set of status codes that are retriable as part of retry options, and the customer can set whichever codes they want retried:
https://github.com/Azure/azure-sdk-for-cpp/blob/3f5c5ae66476f7c77a0ffc0bf1d50f17192166b3/sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp#L93-L99

All other codes won't be retried, so it makes sense to have that as informational. Good call :)